packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md

```diff
@@ -28,10 +29,12 @@
 
 * **Enter your Post Logout Redirect URI** &mdash; provide the complete sign-out redirect URI for your app integration, including the variable names if you're using a per tenant design.
 
+* **Okta requires that all OIDC integrations support multi-tenancy. Do you support this in your implementation?** &mdash; select **Yes** if your customers can set up an OIDC connection between their tenant in your application and multiple external IdPs or even multiple instances inside a single IdP.
+
 * **To configure OIDC, can your customers do it by themselves from your app's UI, or do they need to contact your support team?** &mdash; if a customer needs support to configure your app integration, you need to include support contact information in your configuration guide. We recommend that you build a UI that enables self-service configuration to reduce the setup time for your customers.
 
 * **Link to configuration guide**: Your configuration guide (in either HTML or PDF format) should have step-by-step instructions on how to configure SSO between Okta and your systems. See [Prepare a customer-facing configuration guide](/docs/guides/submit-app/openidconnect/main/#prepare-a-customer-facing-configuration-guide).
```

Comment:
Why have we switched from an mdash to a colon, which continues on the following two questions. Should stick with mdash for consistency thoroughout

Reply:
Good catch - thanks. At the time it made sense, for what it's worth. :)